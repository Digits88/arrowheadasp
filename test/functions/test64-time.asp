<!-- Test Time function.
-->

Time = <%=Time%><br>
