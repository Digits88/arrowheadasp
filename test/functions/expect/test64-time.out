<!-- Test Time function.
-->

Time = ^[0-9]{1,2}:[0-9]{2}:[0-9]{2} (A|P)M$<br>
